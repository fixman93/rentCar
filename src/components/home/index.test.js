import React from 'react'
import Adapter from 'enzyme-adapter-react-16';

import { shallow, configure } from 'enzyme'

import { Home } from './index'

configure({ adapter: new Adapter() });

describe('Homepage', () => {
  const home = shallow(<Home />)
  it('render homepage', () => {
    expect(home).toMatchSnapshot()
  })

  describe('When change select type', () => {
    const city = 'Novi Sad'
    beforeEach(() => {
      home.find('.search-cities').simulate('change', { target: { value: city } })
    })
    it('Update the city in `state`', () => {
      expect(home.state().searcCity.target.value).toEqual(city)
    })
  })

})