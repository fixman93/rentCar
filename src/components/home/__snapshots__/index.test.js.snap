// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage render homepage 1`] = `ShallowWrapper {}`;
